### DISCUSSION TITLE
Potential Connection Error fix

### DISCUSSION THREAD

**Discussion by catears124** [maximmax42/Discord-CustomRP#98] [General]
I'm sure others might have fixed this, but I finally figured out how to fix the connection error. All that I did was delete the %appdata%/CustomRP folder and restart my computer. Once it rebooted, I reinstalled CustomRP and deleted all of the text from the application (id, text, details, boxes) Then it magically worked. This might not work for everyone, but it's worth a try. Anyways, thank you for the app it's really helpful, hope I could help somebody.

**Comment by maximmax42** [comment]
Huh. I'm not sure why that would work.
